test/test_main.cpp

```diff
@@ -1,6 +1,33 @@
 #define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
 #include "catch.hpp"
+#include "../src/time_manager.hpp"
+#include "../src/rtc_time.hpp"
+#include "../src/wrap-hwlib.hpp"
 
 TEST_CASE( "Example Test Case" ) {
     REQUIRE(10 == 10);
 }
+
+TEST_CASE("Time is running") {
+    auto scl = hwlib::target::pin_oc(hwlib::target::pins::scl);
```

Comment:
Clearly, these tests didn't run successfully. More about how to write hwlib tests is commented on this PR.